lang/main-ru.json

```diff
@@ -42,9 +46,31 @@
     "audioOnly": {
         "audioOnly": "Только звук"
     },
+    "blankPage": {
+        "meetingEnded": "Встреча закончилась."
+    },
+    "breakoutRooms": {
+        "actions": {
+            "add": "Добавить комнату отдыха",
+            "autoAssign": "Auto assign to breakout roomsAuto assign to breakout rooms",
```

Comment:
Hrmf. while I am sadly unable to read Dostoyevski in the original, this does not look valid Russian to my untrained eye.